docs/launch-readiness-process.md

```
@@ -0,0 +1,433 @@
+#### 5. CI Job Reference
+```
+ci: security.yml::gitleaks
+```
+- **Workflow file**: without .yml extension
+- **Job name**: exact name from workflow
+- **Used for**: automated checks and scanning
+
+### Multiple Evidence References
+
+If an entry has multiple evidence sources, separate by comma:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)